siriuspy/siriuspy/devices/lillrf.py

```diff
@@ -40,13 +42,31 @@ def amplitude(self, value):
 
     @property
     def phase(self):
-        """."""
+        """Phase."""
         return self['GET_PHASE']
 
     @phase.setter
     def phase(self, value):
         self['SET_PHASE'] = value
 
+    @property
+    def integral(self):
```

Comment:
I think it is better to use `integral_enable`. What do you think?

Reply:
agree!

Reply:
done!